### PR TITLE
Document MongoDB ObjectID functions

### PR DESCRIPTION
**Pull request by is273**
Dear Trino,

Hello, I am Isaac Scafe and recently, I applied for the Technical Writer internship at Starburst. For my writing exercise, I decided to document the ObjectID and ObjectID_timestamp functions as well as updating the links in the list file. This was a fun exercise for me to do as it involved more than just writing technical documents. However, I could not build the docs on my computer as my command prompt kept stating that the config directory did not contain a conf.py. Due to this, I could not test my fixes on my browser. I hope that my documentation meets your standards and that I properly updated the links for the functions.

Doc Screenshot

### CODE REVIEW

**Comment by kokosing** [comment]
Hello, it is nice to meet you :)

Have you followed: https://github.com/trinodb/trino/tree/master/docs#trino-documentation?

Also I advise you to join the slackhttps://trino.io/slack.html so you could get help more interactively.

CC: @mosabua

**Comment by mosabua** [comment]
@is273 please ping me on slack so we can get this sorted out. For starters let me know what operating system you are running, what command you are using, and slack me the error message. We can chat directly or in the #documentation channel

**Comment by martint** [comment]
@cla-bot check

**Comment by is273** [comment]
* Documented objectid and objectid_timestamp functions
* Updated links in list
* Included example use cases

Screenshot from 2021-05-12 22-57-27


docs/build command now works after downloading VM and using Kubuntu.

File: docs/src/main/sphinx/functions.rst [hunk 1]
```diff
@@ -43,4 +43,5 @@ and the :doc:`SQL statement and syntax reference</sql>`.
     System              <functions/system>
     Teradata            <functions/teradata>
     Machine learning    <functions/ml>
+    ObjectID            <functions/objectid>
```
[the remark this answers is not in the document]
Comment (mosabua) [reply docs/src/main/sphinx/functions.rst:46]: also rename the file to mongodb-objectif.rst

File: docs/src/main/sphinx/functions/objectid.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
+===============================
+ObjectID function
```
Comment (mosabua) [inline docs/src/main/sphinx/functions/objectid.rst:2]: MongoDB ObjectID functions

File: docs/src/main/sphinx/functions/objectid.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
+===============================
+ObjectID function
+===============================
+
+.. _objectid:
```
Comment (mosabua) [inline docs/src/main/sphinx/functions/objectid.rst:5]: can you format the whole file to 80 char hard wrap width. Then I can proceed with more detailed input.

**Review by mosabua — CHANGES_REQUESTED** [review]
Lets start with renaming and reformatting. Then we can work on more details.

File: docs/src/main/sphinx/functions/objectid.rst (hunk 3 above)
Comment (is273) [reply docs/src/main/sphinx/functions/objectid.rst]: Changed the name of ObjectID to MongoDB ObjectID and additional changes. Also changed the length of each line to meet the 80 character hard wrap width @mosabua

**Comment by mosabua** [comment]
@is273 I think this looks ready for me to review in more detail now. Are you ok to continue updating if I review this?

**Comment by is273** [comment]
Yes, I'll update the doc after review

**Comment by bitsondatadev** [comment]
:wave: @is273 - this PR has become inactive. If you're still interested in working on it, please let us know, and we can try to get reviewers to help with that.

We're working on closing out old and inactive PRs, so if you're too busy or this has too many merge conflicts to be worth picking back up, we'll be making another pass to close it out in a few weeks.

**Comment by bitsondatadev** [comment]
Gonna close this one out due to inactivity, please reopen if you would like to continue doing work here.

**Comment by mosabua** [comment]
I think we should verify if this information is currently missing from the docs and then potentially pick this up. Can you run with this @bitsondatadev ?

**Comment by bitsondatadev** [comment]
This was verified by @ebyhr as not necessary.

**Pull request closed without merging**